jfr-streaming/src/parallelGcTest/java/io/opentelemetry/contrib/jfr/metrics/PsGcDurationMetricTest.java

```diff
@@ -18,35 +18,16 @@
 import org.assertj.core.api.ThrowingConsumer;
 import org.junit.jupiter.api.Test;
 
-class GCDurationMetricTest extends AbstractMetricsTest {
+class PsGcDurationMetricTest extends AbstractMetricsTest {
   static class AttributeCheck implements ThrowingConsumer<PointData> {
```

Comment:
I don't think having a separate class makes the test code more readable. Let's inline this.